Marlin/src/inc/SanityCheck.h

```diff
@@ -21,6 +21,8 @@
  */
 #pragma once
 
+#include "../pins/pins_lcd.h"   // Needed to define LED and NEOPIXEL pins on some displays
```

Comment:
This is already guaranteed to be included before this file.

Reply:
removed